migration-engine/connectors/sql-migration-connector/src/sql_destructive_changes_checker/destructive_check_plan.rs

```diff
@@ -153,3 +165,9 @@ async fn count_values_in_column(
 
     Ok(values_count)
 }
+
+/// The time we let the destructive check database inspection queries to run, in total. Currently
+/// hardcoded to 1 minute.
+fn destructive_check_timeout_duration() -> Duration {
+    Duration::from_secs(60)
+}
```

Comment:
We should be able to [const-initialize this instead](https://doc.rust-lang.org/std/time/struct.Duration.html#method.from_secs). I would expect this to be a constant at the top of the file instead:

```rust
/// The time we let the destructive check database inspection queries to run, in total. Currently
/// hardcoded to 1 minute.
const DESTRUCTIVE_TIMEOUT_DURATION: Duration = Duration::from_secs(60);
```